webapp/src/pages/LedgersOverview.js

```diff
@@ -1,78 +1,150 @@
 import api from "../api";
+import ForwardBackPagination from "../components/ForwardBackPagination";
 import Money from "../components/Money";
 import PageLoader from "../components/PageLoader";
 import TopNav from "../components/TopNav";
+import LedgerItemModal from "../components/ledger/LedgerItemModal";
 import useAsyncFetch from "../shared/react/useAsyncFetch";
+import useListQueryControls from "../shared/react/useListQueryControls";
+import relativeUrl from "../shared/relativeUrl";
+import setUrlPart from "../shared/setUrlPart";
+import clsx from "clsx";
+import dayjs from "dayjs";
 import i18next from "i18next";
 import _ from "lodash";
 import React from "react";
 import Button from "react-bootstrap/Button";
 import Container from "react-bootstrap/Container";
-import Navbar from "react-bootstrap/Navbar";
-import Stack from "react-bootstrap/Stack";
-import { useNavigate } from "react-router-dom";
+import Table from "react-bootstrap/Table";
+import { useLocation, useNavigate } from "react-router-dom";
 
-const LedgersOverview = () => {
-  const { state: dashboard, loading: dashboardLoading } = useAsyncFetch(api.ledgers, {
+export default function LedgersOverview() {
```

Comment:
@DeeTheDev this was a pretty big change, please read it over and lmk what questions you have.